utils/tini/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tini
+PKG_VERSION:=0.18.0
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/krallin/tini/tar.gz/v${PKG_VERSION}?
+PKG_HASH:=1097675352d6317b547e73f9dc7c6839fd0bb0d96dafc2e5c95506bb324049a2
+
+PKG_MAINTAINER:=Gerard Ryan <G.M0N3Y.2503@gmail.com>
+
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/tini
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=simplest init utility
+  URL:=https://github.com/krallin/tini
+  DEPENDS:=
+endef
+
+define Package/tini/description
+A tiny but valid init process for containers.
+endef
+
+TARGET_CFLAGS+= \
+  -DPR_SET_CHILD_SUBREAPER=36 \
+  -DPR_GET_CHILD_SUBREAPER=37
```

Comment:
Likely this part can be omitted.  [prctl(2)](http://man7.org/linux/man-pages/man2/prctl.2.html) says these were introduced in kernel 3.14.  Kernels used by OpenWrt should already have this for quite some time.

Reply:
I wasn't sure what kernel OpenWrt was at but it looks like you're right, these are added in the current version.